iOS/DuckDuckGo/AutofillContentScopeFeatureToggles.swift

```diff
@@ -41,6 +41,8 @@ extension ContentScopeFeatureToggles {
             unknownUsernameCategorization: featureFlagger.isFeatureOn(.autofillUnknownUsernameCategorization) && isAutofillEnabledInSettings,
             partialFormSaves: featureFlagger.isFeatureOn(.autofillPartialFormSaves) && isAutofillEnabledInSettings,
             passwordVariantCategorization: featureFlagger.isFeatureOn(.autofillPasswordVariantCategorization) && isAutofillEnabledInSettings,
-            inputFocusApi: featureFlagger.isFeatureOn(.autofillCreditCards) && isCreditCardAutofillEnabledInSettings)
+            inputFocusApi: featureFlagger.isFeatureOn(.autofillCreditCards) && isCreditCardAutofillEnabledInSettings,
+            autocompleteAttributeSupport: isAutofillEnabledInSettings
```

Comment:
Instead of using `isAutofillEnabledInSettings` here, let’s set this up to use your new remote feature flag 🙂 To do that you’ll need to add the flag [here](https://github.com/duckduckgo/apple-browsers/blob/17f69a83fb851cfe5b765eb3cbdc137fd217c9b5/SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift#L136) to the AutofillSubfeature, and then update [iOS](https://github.com/duckduckgo/apple-browsers/blob/matteo/add-autocompleteAttributeSupport-subfeature-flag/iOS/Core/FeatureFlag.swift) to set the flag to be `.remoteReleasable` etc.